Shared/Supporting Files/Views/SampleDetailView.swift

```diff
@@ -109,3 +112,10 @@ struct SampleDetailView: View {
 extension SampleDetailView: Identifiable {
     var id: String { sample.nameInUpperCamelCase }
 }
+
+private extension URL {
+    /// The URL to the Samples sub-directory on GitHub's main branch.
+    static let samplesDirectory = URL(
+        string: "https://github.com/Esri/arcgis-maps-sdk-swift-samples"
+    )!.appendingPathComponent("tree/main/Shared/Samples")
```

Comment:
No need to create two URLs.

```suggestion
        string: "https://github.com/Esri/arcgis-maps-sdk-swift-samples/tree/main/Shared/Samples"
    )!
```